azurelinuxagent/distro/suse/rdma.py

```diff
@@ -34,11 +34,14 @@ def install_driver(self):
             error_msg += 'no driver will be installed.'
             logger.error(error_msg)
             return
-        zypper_install = 'zypper -n in'
-        zypper_search = 'zypper se -s'
+        zypper_install = 'zypper -n in %s'
+        zypper_remove = 'zypper -n rm %s'
+        zypper_search = 'zypper se -s %s'
```

Comment:
In the old code we had:

     self.command_to_get_rdma_package_info = self.rpm_path + " -q " + self.rdma_package_name

which would just return the package name like `microsoft-hyper-v-rdma-4.1.0.142-20160323.x86_64`. Isn't this a better fit to determine the installed package?